lms/static/certificates/sass/_components.scss

```diff
@@ -270,14 +270,11 @@
 
     // organizations
     .accomplishment-orgs {
+        @extend %rendering-section;
 
         .list-orgs {
             @extend %list-unstyled;
         }
-
-        .organization-logo {
-            @include filter(grayscale(100%));
```

Comment:
NOTE: some browsers were not handling this filter well in print context, so I just removed it for all cases - logos will appear full color.